pages/challenge.js

```diff
@@ -0,0 +1,154 @@
+    this.state = {
+      names: '',
+    };
+  }
+
+  componentDidMount() {
+    const { currentNames } = this.state;
+    if (!currentNames) {
+      axios.get(namesFile).then(response => {
+        const names = response.data;
+        this.setState({ names });
+      });
+    }
+  }
+
+  render() {
+    const { names } = this.state;
+    return (
+      <div>
+        <Section title="Operation Code Challenge" theme="white" className={styles.displayBlock}>
+          <div>
+            Welcome to the Operation Code challenge! The goal of this challenge is to get you to
+            easily commit your first change to a program, see the results of the change, and leave
+            your mark on Operation Code itself! To do this we&apos;re going to take a look at a
+            source code repository, clone the repository, make a change to a file and finally create
+            a pull request.
+          </div>
+          <br />
+          <h6 className={styles.centerText}>Let&apos;s get started!</h6>
+          <br />
+          <ol className={styles.instructionList}>
+            <li>
+              Firstly, visit our GitHub repository for this page via this link{' '}
+              <OutboundLink analyticsEventLabel={`${RepoLink}`} href={`${RepoLink}`}>
+                {' '}
+              </OutboundLink>
+              . GitHub is a website dedicated to hosting source code. That&apos;s right -&nbsp; the
+              code for this website is publically available! Take a moment to explore GitHub. You
+              can see the code for this specific page via&nbsp; this link{' '}
+              <OutboundLink analyticsEventLabel={`${ChallengeLink}`} href={`${ChallengeLink}`}>
+                {' '}
+              </OutboundLink>
+              .
+            </li>
+            <li>
+              Secondly, fork the repository. Forking a repository takes a snapshot of the repo and
+              places that snapshot into your personal area. This allows anyone to make changes to
+              any project and easily contribute them back. Visit this link{' '}
+              <OutboundLink analyticsEventLabel={`${RepoLink}`} href={`${RepoLink}`}>
+                {' '}
+              </OutboundLink>
+              &nbsp; and click on this button near the top-right corner of the screen:&nbsp;
+              <img src={ForkButton} alt="the GitHub Fork Button" width="65px" /> In a few moments,
+              you will be redirected to your own copy of this website&apos;s source code.
+              <br />
+              <br />
+              <b>Congratulations! You now have your very own personal copy of our website!</b>
+            </li>
+            <li>
+              Now that you have a fork of the &quot;repo&quot;, it&apos;s time to edit the necessary
+              file to add your name to the list below! Inside the <code>/static</code> folder ,
+              click on the <code>operationcode_challenge</code> directory and click on the file
```

Comment:
Good catch to update the content! In case you were curious, we separate the names from the code so that Git history reflects well. We don't want to need to revert code changes and then rebase all of the names.